src/features/decorators/bonusDecorator.ts

```diff
@@ -1,44 +1,9 @@
-import { Replace, OptionsObject } from "./feature";
-import Decimal, { DecimalSource } from "util/bignum";
-import { Computable, GetComputableType, processComputable, ProcessedComputable } from "util/computed";
-import { Persistent, State } from "game/persistence";
-import { computed, Ref, unref } from "vue";
+import { Replace } from "features/feature";
+import Decimal, { DecimalSource } from "lib/break_eternity";
```

Comment:
Please change this back to util/bignum

Reply:
Heh, guess that's what I get for just letting it mass-import everything

Reply:
Trust me, I've made this same mistake so many times. I wish I had a better way to hint to the IDE what the default export is expected to be called